setup.cfg

```diff
@@ -29,7 +29,9 @@ source=dask_ml
 addopts = -rsx -v --durations=10
 minversion = 3.2
 xfail_strict = true
+junit_family = xunit2
 filterwarnings =
+     error:::dask_ml[.*]
      error:::sklearn[.*]
      # TODO: Remove once https://github.com/scikit-learn/scikit-learn/pull/14948 is merged
      ignore:The sklearn.cluster
```

Comment:
https://github.com/scikit-learn/scikit-learn/pull/14948 is merged and part `scikitlearn==0.22`. Can this line be removed?

Reply:
Seems like it, thanks :)